### PR TITLE
Update python-psutil to 5.7.2

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@evgeni Do you know if the above is true? And/or have any suggestions given your recent playing with python packaging?

**Comment by evgeni** [comment]
Setuptools on EL7 could be a Snakeasaurus. So yeah, pretty much possible.

**Comment by ekohl** [comment]
It's just a helper for pip to know how to select it, but that's not very relevant for RPMs since that can also ensure the correct Python version so it's safe to strip it out. See https://github.com/giampaolo/psutil/commit/f7171c45d7cfc1ce68baa7cd0afdaa94e28305a5 as well.

**Comment by ehelms** [comment]
This ended up being something with the format of `extras_requires` which is only used to specify test requires. I went the route of patching this to an empty hash similar to how it was in past releases so that `setup.py` could proceed on EL7.

File: packages/client/python-psutil/python-psutil.spec [hunk 1]
```diff
@@ -57,7 +53,8 @@ ifconfig, nice, ionice, iostat, iotop, uptime, pidof, tty, who, taskset, pmap.
 
 
 %prep
-%autosetup -p0 -n %{srcname}-release-%{version}
+%setup -q -n %{srcname}-release-%{version}
+%patch0 -p1
```
Comment (evgeni) [inline packages/client/python-psutil/python-psutil.spec:57]: Technically you could have left the %autosetup here, with -p1. But no hard preference either way from me.

Comment (ehelms) [reply packages/client/python-psutil/python-psutil.spec]: Sold!

**Review by evgeni — APPROVED** [review]

**Pull request merged**